app/src/main/java/org/simple/clinic/forgotpin/confirmpin/ForgotPinConfirmPinScreen.kt

```diff
@@ -48,6 +49,10 @@ class ForgotPinConfirmPinScreen(context: Context, attributeSet: AttributeSet?) :
       RxView.clicks(facilityNameTextView)
           .map { ForgotPinConfirmPinScreenFacilityClicked }
 
+  private fun backClicks(): Observable<UiEvent> =
+      RxView.clicks(backButton)
+          .map { ForgotPinConfirmPinScreenBackClicked }
+
```

Comment:
Handle back button clicks in the UI directly please. Tests are not worth having for this. :)

Reply:
Will update in the next PR.